src/PowerShellEditorServices/Services/Analysis/AnalysisService.cs

```diff
@@ -286,22 +310,22 @@ private bool TryFindSettingsFile(out string settingsFilePath)
         {
             string configuredPath = _configurationService.CurrentSettings.ScriptAnalysis.SettingsPath;
 
-            if (!string.IsNullOrEmpty(configuredPath))
+            if (string.IsNullOrEmpty(configuredPath))
             {
-                settingsFilePath = _workplaceService.ResolveWorkspacePath(configuredPath);
+                settingsFilePath = null;
+                return false;
+            }
 
-                if (settingsFilePath == null)
-                {
-                    _logger.LogError($"Unable to find PSSA settings file at '{configuredPath}'. Loading default rules.");
-                }
+            settingsFilePath = _workplaceService.ResolveWorkspacePath(configuredPath);
 
-                return settingsFilePath != null;
+            if (settingsFilePath == null
+                || !File.Exists(settingsFilePath))
+            {
+                _logger.LogError($"Unable to find PSSA settings file at '{configuredPath}'. Loading default rules.");
```

Comment:
should you set the `settingsFilePath` to `null` here just to be clear that this method returns `false and null` instead of `false and possibly null possilbly a path that doesn't exist`